test/inspec/spec_helper.rb

```diff
@@ -8,5 +8,6 @@ def init_conf
     '::packer_build_name' => Facter.value(:packer_build_name),
     '::os' => Facter.value(:os)
   }
-  @hiera = Hiera.new(config: Facter.value(:packer_staging_dir) + '/hiera.yaml')
+  # TODO: Fix this when inspec has a path from hiera 3 to 5
```

Comment:
Please add a link to the discussion thread that explains why hiera gem is deprecated, that thread also has a good pointer to look at rspec-puppet.